### DISCUSSION TITLE
Font size for file name in torrent list

### DISCUSSION THREAD

**Discussion by gennaios** [jesec/flood#203] [General]
Hello,

Thanks for continuing the work.

Since I began using it, I've thought the font size for the file names in a listing are too large. I think they should match the size of the other stats (numbers) in the same row. Such is how tables appear in native torrent apps and perhaps even most places that use tables with columns and headers in native desktop or web apps.

**Comment by jesec** [comment]
It is just **bold**. This is part of the design.

**Reply by gennaios** [reply]
Hello. I understand it’s part of the design. Though I consider it out of place and maybe not best. Perhaps it was done before you took over (if you were not involved). How it looks, do other tables elsewhere e.g. in desktop apps like file managers also do such? I think it looks better without. :) I’ve modified the CSS in my copy and indeed I think it is nicer.
